### DISCUSSION TITLE
[@types/web-bluetooth] BluetoothUUID not included

### DISCUSSION THREAD

**Discussion by grimsteel** [DefinitelyTyped/DefinitelyTyped#65131] [Issues with a @types package]
Type definitions for the BluetoothUUID interface aren't included

It only includes 4 methods, so I'd could try and do a PR for this myself if that's okay.

[MDN Reference](https://developer.mozilla.org/en-US/docs/Web/API/BluetoothUUID)

[a comment by an automated account is not shown]

**Comment by thegecko** [comment]
These are currently marked as experimental on MDN, do you know if they are implemented in chrome or edge?

**Reply by grimsteel** [reply]
The BluetoothUUID interface has similar browser compatibility to the Bluetooth interface. It's implemented in Chrome/Edge. I think the entire Web Bluetooth API is marked as experimental on MDN 😉

**Comment by thegecko** [comment]
Ok, I see it exists on the global (window) object alongside `navigator`. I can confirm it exists in Chrome.

Please feel free to create a PR to add it to these types.
